pnc-model/src/main/java/org/jboss/pnc/model/BuildRecord.java

```diff
@@ -57,10 +59,13 @@
 
     private String patchesUrl;
 
+    @NotNull
     private Timestamp startTime;
 
+    @NotNull
     private Timestamp endTime;
 
+    @NotNull
     @ManyToOne
     private User user;
 
```

Comment:
Since we are going to use Envers for the BuildConfiguration parameters, ok for now not to mark as NotNull the "name" field